mmtrack/datasets/pipelines/transforms.py

```diff
@@ -798,3 +798,40 @@ def __repr__(self):
         repr_str += f'{(self.saturation_lower, self.saturation_upper)},\n'
         repr_str += f'hue_delta={self.hue_delta})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class SeqGrayAug(object):
```

Comment:
we can put the class in the front of [SeqShiftScaleAug](https://github.com/open-mmlab/mmtracking/blob/master/mmtrack/datasets/pipelines/transforms.py#L137)